airbyte-integrations/connectors/source-postgres/src/test/java/io/airbyte/integrations/source/postgres/PostgresJdbcStandardSourceTest.java

```diff
@@ -47,6 +48,11 @@
   @BeforeAll
   static void init() {
     PSQL_DB = new PostgreSQLContainer<>("postgres:13-alpine");
+//        .withFileSystemBind("/tmp/airbyte_crt/server.crt", "/var/lib/postgresql/server.crt",
+//            BindMode.READ_WRITE)
+//        .withFileSystemBind("/tmp/airbyte_crt/server.key", "/var/lib/postgresql/server.key", BindMode.READ_WRITE)
+//        .withCommand("postgres -c ssl=on -c ssl_cert_file=/var/lib/postgresql/server.crt -c ssl_key_file=/var/lib/postgresql/server.key");
```

Comment:
(I kept this to discuss tests) Is possible to add tests using test container and Postgres.

Reply:
It'd be great to have a test at least for the check connection to something that's configured like this. We could either commit some key files or generate them on the fly.

Reply:
For Postgres is easy to enable SSL but for other databases is not so trivial